docs/HowToFAQ/ERROR_ProverError1.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Prover error: Unexpected prover response (getting info about 'unknown' response): (:reason-unknown 'Overflow encountered when expanding old_vector')"
+---
+
+This error is caused by a Z3 bug. The best you can do is to try to change the verification condition sent to Z3 by splitting it up, using various options and attributes like `{:split_here}`, `{:focus}`, `/vcsSplitOnEveryAssert`, `/vcsMaxSplits`, and `/randomSeed`.
```

Comment:
I'd say it's a Dafny bug since we're using a too old Z3 version ^,^

Also towards the user I think it's easier to talk in terms of Dafny than the underlying technology used.

Reply:
Reworded. And hopefully the problem will be moot with a new Z3 soon.